plugins/trace.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: trace
+spec:
+  version: "v0.3.4"
+  homepage: https://github.com/iovisor/kubectl-trace
+  shortDescription: Schedule bpftrace programs on your kubernetes cluster using the kubectl.
+  caveats: |
+    kubectl trace is now installed! 
+
+    Examples:
```

Comment:
I think these examples would fit better in a `description`, of as @.ahmetb said in the `--help` output.

Also, it's usually nice to put a rough description of the technical details into the description. Just to give the users an idea of what is going on when invoking the plugin.

Reply:
That's exactly what I wanted thanks @corneliusweig !

Reply:
I think you gave a nice collection of examples which are well suited for people familiar with `kubectl-trace`. However, I'd recommend to keep the description on a more conceptual level and give the (potential) users an idea of the key concepts of `kubectl-trace`. Also, you should try to explain why and in which situations users could benefit from this plugin.
Further, you should explain how `trace` does its magic, for example:

> `trace` loads bpf programs into the host kernels on the target node and uses these programs to collect metrics.

(not sure if that's totally accurate, but you get the point, right?)

In essence, the description is a bit of a sales pitch for potential users. So use it wisely :)